.travis.yml

```diff
@@ -1,6 +1,7 @@
 sudo: false
 language: node_js
 node_js:
-  - stable
+  - 7
```

Comment:
`node` is the name for the latest stable version. I think I prefer this to 7 given it won't go into LTS.